public/locales/en.json

```diff
@@ -410,6 +410,32 @@
     "overview": "Overview",
     "proposer": "Proposer"
   },
+  "IDOPage": {
+    "unlockVotes": "Unlock Votes",
```

Comment:
1. we need to add `IDOPage` for all languages.
2. do we need all these keys ? lets remove unnecessary fields.

Reply:
1. Added the IDOPage for all languages with translations
2. Removed unnecessary keys